functions.js

```diff
@@ -1 +1,102 @@
-// TODO: add your solutions here!
+
+//1.build concentation  (return)
+function combineWords(word1, word2) {
+
+    var result === word1+word2;
+    console.log(result);
+    return result;
+}
+combineWords ("dog","house")
+
+
+//2.repeat phrase  (display)
+function repeatPhrase(phrase, n){
+    for (var i===0; i< n ; i++){
+    console.log(phrase, n);
+}
+
+repeatPhrase("Hello", 5);
+
+
+//3.power function  (return)
+function toTheNthPower(number,power){
+    var result = 1
+    for (var i = 0; i < power ,i ++);
+}
+
```

Comment:
You are missing a code block after you instantiate your for loop.
for(var i = 0; i < power ; i++){
  result *= number
}